notebooks/02_model/vowpal_wabbit_deep_dive.ipynb

```diff
@@ -0,0 +1,737 @@
+       "      <td>2</td>\n",
+       "      <td>196</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>3</td>\n",
+       "      <td>1259</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>978298296</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>13</th>\n",
+       "      <td>2</td>\n",
+       "      <td>201</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>3</td>\n",
+       "      <td>260</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>978297512</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>14</th>\n",
+       "      <td>2</td>\n",
+       "      <td>202</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>3</td>\n",
+       "      <td>2858</td>\n",
+       "      <td>4.0</td>\n",
+       "      <td>978297039</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    level_0  level_1  prediction  userID  itemID  rating  timestamp\n",
+       "0         0        0         5.0       1    1193     5.0  978300760\n",
+       "1         0        2         5.0       1     914     3.0  978301968\n",
+       "2         0        3         5.0       1    3408     4.0  978300275\n",
+       "3         0        4         5.0       1    2355     5.0  978824291\n",
+       "4         0        5         5.0       1    1197     3.0  978302268\n",
+       "5         1       53         5.0       2    1357     5.0  978298709\n",
+       "6         1       54         5.0       2    3068     4.0  978299000\n",
+       "7         1       55         5.0       2    1537     4.0  978299620\n",
+       "8         1       57         5.0       2    2194     4.0  978299297\n",
+       "9         1       61         5.0       2    1103     3.0  978298905\n",
+       "10        2      193         5.0       3    2997     3.0  978298147\n",
+       "11        2      195         5.0       3    1291     4.0  978297600\n",
+       "12        2      196         5.0       3    1259     5.0  978298296\n",
+       "13        2      201         5.0       3     260     5.0  978297512\n",
+       "14        2      202         5.0       3    2858     4.0  978297039"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# load predictions and filter to test set\n",
+    "test_users = [1, 2, 3]\n",
+    "pred_data = pd.read_csv(prediction_path, delim_whitespace=True, names=['prediction'], index_col=1).join(df)\n",
+    "test_user_data = pred_data[pred_data['userID'].isin(test_users)]\n",
+    "\n",
+    "get_top_k_items(test_user_data, col_rating='prediction', k=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h3>Cleanup</h3>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tmpdir.cleanup()"
```

Comment:
I think a conclusion would be great. Maybe as @eisber suggested to have a final table with all the results

Reply:
ok